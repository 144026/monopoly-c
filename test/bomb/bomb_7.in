preset user AQ
preset option mskip on
preset userloc A 25 0
preset userloc Q 26 0
preset gift A bomb 3
# do not allow bomb on special node: item house
bomb 3
step 1
n
step 3
n
dump