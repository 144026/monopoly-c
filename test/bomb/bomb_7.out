user AQ
fund A 10000
fund Q 10000
credit A 0
credit Q 0
userloc A 26 0
userloc Q 29 0
gift A bomb 3
nextuser A